### DISCUSSION TITLE
Alternatives as an option

### DISCUSSION THREAD

**Discussion by hadmut** [erikhuda/thor#859] [Ideas]
Hi,

would be nice to have an option type to choose one of a given set opf alternatives. e.g.

method_option :fruit, type: :selection, values: [ :apple, :orange, :banana ]

and then call a program with
examplebin --banana

regards

**Comment by p8** [comment]
hi @hadmut,
You should be able to do the following:
```ruby
  method_option :fruit, type: :string, enum: %w(apple orange banana)
```
